operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -1,263 +1,379 @@
+      ncn-m001# kubectl -n kube-system get secret sealed-secrets-key -o jsonpath='{.data.tls\.crt}' | base64 -d - > certs/sealed_secrets.crt
+      ncn-m001# kubectl -n kube-system get secret sealed-secrets-key -o jsonpath='{.data.tls\.key}' | base64 -d - > certs/sealed_secrets.key
+      ```
+
+  > **NOTE:** All subsequent steps of this procedure should be performed within the `/root/site-init` directory created in this step.
+
+2. Generate Sealed Secrets.
+   
+   Secrets are stored in `customizations.yaml` as `SealedSecret` resources 
+   (encrypted secrets), which are deployed by specific charts and decrypted by the
+   Sealed Secrets operator. But first, those Secrets must be seeded generated and
+   encrypted.
+
+   The LDAP server CA certificate goes into SealedSecret.
+
+   1. Load the `zeromq` container image required by Sealed Secret Generators:
+
+      > **NOTE:** A properly configured Docker or Podman environment is required.
+
+      ```bash
+      linux# /mnt/pitdata/${CSM_RELEASE}/hack/load-container-image.sh dtr.dev.cray.com/zeromq/zeromq:v4.0.5
+      ```
+
+   2. Re-encrypt existing secrets:
+
+      ```bash
+      linux# /mnt/pitdata/prep/site-init/utils/secrets-reencrypt.sh /mnt/pitdata/prep/site-init/customizations.yaml \
+      /mnt/pitdata/prep/site-init/certs/sealed_secrets.key /mnt/pitdata/prep/site-init/certs/sealed_secrets.crt
+      ```
+
+   3. Generate secrets:
+
+      ```bash
+      linux# /mnt/pitdata/prep/site-init/utils/secrets-seed-customizations.sh \
+      /mnt/pitdata/prep/site-init/customizations.yaml
+      ```
+
+      Expected output looks similar to:
+
+      ```bash
+      Creating Sealed Secret keycloak-certs
+      Generating type static_b64...
+      Creating Sealed Secret keycloak-master-admin-auth
+      Generating type static...
+      Generating type static...
+      Generating type randstr...
+      Generating type static...
+      Creating Sealed Secret cray_reds_credentials
+      Generating type static...
+      Generating type static...
+      Creating Sealed Secret cray_meds_credentials
+      Generating type static...
+      Creating Sealed Secret cray_hms_rts_credentials
+      Generating type static...
+      Generating type static...
+      Creating Sealed Secret vcs-user-credentials
+      Generating type randstr...
+      Generating type static...
+      Creating Sealed Secret generated-platform-ca-1
+      Generating type platform_ca...
+      Creating Sealed Secret pals-config
+      Generating type zmq_curve...
+      Generating type zmq_curve...
+      Creating Sealed Secret munge-secret
+      Generating type randstr...
+      Creating Sealed Secret slurmdb-secret
+      Generating type static...
+      Generating type static...
+      Generating type randstr...
+      Generating type randstr...
+      Creating Sealed Secret keycloak-users-localize
+      Generating type static...
+      ```
+   
+3.   Update the LDAP settings in the customizations.yaml file.
```

Comment:
this doc should be consistent about using customizations.yaml or `customizations.yaml`.